canopen/objectdictionary/eds.py

```diff
@@ -40,7 +41,7 @@ def import_eds(source, node_id):
         match = re.match(r"^[0-9A-Fa-f]{4}$", section)
         if match is not None:
             index = int(section, 16)
-            name = eds.get(section, "ParameterName")
+            name = eds.get(section, "ParameterName", raw=True)
```

Comment:
Doesn't work in Python 2.7 (see [Travis](https://travis-ci.org/christiansandberg/canopen/jobs/332983799)).

Reply:
I'm struggling to see why this do not work. I have tested this on my Mac with python 2.7.10. The ConfigParser package is:

pip show configparser
---
Metadata-Version: 1.1
Name: configparser
Version: 3.5.0

Also according to the ConfigParser documentation it seems to indicate that the raw options should be supported:
https://docs.python.org/2/library/configparser.html

Any suggestions?

Reply:
For Python 2.7 I used RawConfigParser so it may just be to switch to ConfigParser.

Reply:
That explained it. I did change to use the RawConfigParser independent of the module that is imported.